components/MerchGeneral/Merch/HowItWorks/how-it-works.tsx

```diff
@@ -1 +1,43 @@
-export const HowItWorks = () => <h1>How it works</h1>;
+import { urlForImage } from '@/lib/sanity';
+import Image from 'next/image';
+import { ImagesListItem } from '@/types';
+
+type Props = {
+  howItWorksImages: ImagesListItem[];
+  howItWorksBlock: string;
+};
```

Comment:
It's an interface, not a type

Reply:
I think it is a type, all other pages use type for this case

Reply:
> I think it is a type, all other pages use type for this case

So all the other pages are use this wrong